docs/packages/pkg/flex.rst

```diff
@@ -0,0 +1,32 @@
+.. spelling::
+
+    flex
+
+.. index:: compiler ; flex
+
+.. _pkg.flex:
+
+flex
+=====
+
+-  `Official <https://github.com/westes/flex/>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/flex/CMakeLists.txt>`__
+-  Added by `Isaac Hier <https://github.com/isaachier>`__ (`pr-N <https://github.com/ruslo/hunter/pull/N>`__)
+
+.. code-block:: cmake
+
+    # Based on https://cmake.org/cmake/help/v3.0/module/FindFLEX.html.
```

Comment:
I think it make sense to create minimal example with `find_package(FLEX)` + `hunter_add_package(flex)` and without Bison. You can create two blocks with CMake code if you think that Bison + Flex example is useful.